assets/cms/scss/panels/_shared.scss

```diff
@@ -366,6 +375,10 @@ html.ccm-panel-ready div.ccm-page.ccm-panel-detail-disable-transition {
   transition: none;
 }
 
+div.ccm-panel-dashboard-footer {
+  margin:20px 0;
```

Comment:
Can you fix the formatting here

Reply:
I'll just make this change post merge

Reply:
https://github.com/concrete5/bedrock/commit/63a3a3cdc72bbf735b8af5931e8af18a68d4967f

Reply:
Thanks @KorvinSzanto !